icu4c/source/test/intltest/calcasts.cpp

```diff
@@ -499,5 +502,85 @@ void CalendarCaseTest::Ethiopic() {
     delete c;
 }
 
+void CalendarCaseTest::Tibetan() {
+
+    //Testing Phugpa calculations 
+
+    static const TestCase tests[] = {
+
+    // Test cases to make sure that computeTime and computeFields
+    // work properly for a wide range of data in the Tibetan calendar.
+    //
+    // Tests come from data produced by Edward Henning on
+    // http://kalacakra.org/calendar/tiblist.htm
+    //
+    // Julian Day   Era Year     Month         Day  WkDay Hour Min Sec
+       {2458165.5,  17,    32,         1,        1,  FRI,   0,  0,  0}, // Gregorian: 16/02/2018 - not a leap month - not a leap day - Losar
+       {2432757.5,  16,    22,         6,       19,  SUN,   0,  0,  0}, // Gregorian: 25/07/1948 - not a leap month - not a leap day - Shakyamuni Buddha Day New Moon
+       {2361500.5,  13,     7,         5,       84,  WED,   0,  0,  0}, // Gregorian: 20/06/1753 - not a leap month - leap day
```

Comment:
could you add a test entry for the date BEFORE and AFTER that leap day too?

Reply:
Thanks! Just did, I realized that the test doesn't really represent what I anticipated would be the right answer... this new test does